src/Card/constants.ts

```diff
@@ -0,0 +1 @@
+export const CARD_HEIGHT = '76px';
```

Comment:
I noticed this wasn't a constant to the theme spacing, I forgot why it was an issue. I personally changed it to 80.

Reply:
I took this from the designs directly. We can change it, but maybe it should be the same for all "cards" ?

Reply:
@pyphilia what should be settle on ?